content/en/continuous_integration/intelligent_test_runner/setup/javascript.md

```diff
@@ -92,16 +89,126 @@ NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app DD_CIV
 
 ### Cypress
 
-For Intelligent Test Runner for Cypress to work, you must instrument your web application with code coverage. For more information about enabling code coverage, see the [Cypress documentation][3]. 
+For Intelligent Test Runner for Cypress to work, you must instrument your web application with code coverage. For more information about enabling code coverage, see the [Cypress documentation][2].
 
 To check that you've successfully enabled code coverage, navigate to your web app with Cypress and check the `window.__coverage__` global variable. This is what `dd-trace` uses to collect code coverage for Intelligent Test Runner.
 
+## Inconsistent test durations
+
+In some frameworks, such as `jest`, there are cache mechanisms that make tests faster after other tests have run (see [jest cache][4] docs). If Intelligent Test Runner is skipping all but a few test files, these suites might run slower than they usually do. This is because they run with a colder cache. Regardless of this, total execution time for your test command should still be reduced.
+
+## Disabling skipping for specific tests
+
+You can override the Intelligent Test Runner's behavior and prevent specific tests from being skipped. These tests are referred to as unskippable tests.
+
+### Why make tests unskippable?
+
+The Intelligent Test Runner uses code coverage data to determine whether or not tests should be skipped. In some cases, this data may not be sufficient to make this determination.
+
+Examples include:
+
+* Tests that read data from text files
+* Tests that interact with APIs outside of the code being tested (such as remote REST APIs)
+
+Designating tests as unskippable ensures that the Intelligent Test Runner runs them regardless of coverage data.
+
+### Marking tests as unskippable
+
+{{< tabs >}}
+{{% tab "Jest/Mocha/Cypress" %}}
+You can use the following docblock at the top of your test file to mark a suite as unskippable. This prevents any of the tests defined in the test file from being skipped by Intelligent Test Runner. This is similar to jest's [`testEnvironmentOptions`][1].
+
+```javascript
+/**
+ * @datadog {"unskippable": true}
+ */
+
+// your tests continue normally
```

Comment:
This just seems weird to me. The file continues normally? Tests continue to run normally?

Is this necessary?

Reply:
I'll remove it. I was on the fence about it